mautic/mautic
# Added support for global categories

| Q | A |
| --- | --- |
| Bug fix? |  |
| New feature? | y |
| Related user documentation PR URL | todo |
| Related developer documentation PR URL |  |
| Issues addressed (#s or URLs) |  |
| BC breaks? | n |
| Deprecations? | n |
#### Description:

This simply adds support for global categories that can be used across different bundles/entities.
#### Steps to test this PR:
1. Create a new category and choose "Global" as the type.
2. Create a new email, asset, etc and the new Global category should be available along with other entity specific categories. 
3. Permissions for global cats are managed by Categories - Permissions
